### DISCUSSION TITLE
[Feature] Multi-tagged person vs. untagged person

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#6005] [Feature Request]
### The feature

Currently, there is no way to distinguish whether a face has not yet been recognized as a face, or if the face is actually mistakenly associated with another person already present in the picture. Except if you start editing the faces from the side pane.

Example: My son and my daughter are present on the picture. But the side pane shows only the facegroup of my son.
If I don't mouse over or edit facegroups, then I don't know whether my daughter has not been recognized or whether my son is double-tagged instead of her.
If I mouseover, I see that my son is double-tagged.

Proposed feature:
- Add a bubble at the bottom of the thumbnail of the face (in the side pane) to display the number of times the face appears on the pic. 
Result: I immediately know that there is a problem with that pic.

Basically, I'm at the point of using Immich where I go into details of the wrong classification, and the hardest part is to find the pictures that really has one person (my son in the example), but still has misclassified the other person as my son...
My eyes can't find these pictures easily.

### Platform

- [X] Server
- [X] Web
- [X] Mobile
